### ISSUE TITLE
`op-deployer` triggers security warning on macos

### ISSUE THREAD

**Issue by tessr** [ethereum-optimism/optimism#13112]
When I install `op-deployer` from the [release page](https://github.com/ethereum-optimism/optimism/releases), macOS warns me that the software can't be verified. There isn't an obvious way to override this. I have to go into my [system settings](https://support.apple.com/guide/mac-help/open-a-mac-app-from-an-unknown-developer-mh40616/mac) to get `op-deployer` to run.

I don't run into this problem when building `op-deployer` from source. 

![Image](https://github.com/user-attachments/assets/4c71ab78-8a3f-4556-bd48-61c37c8e9640)

This is low-priority but it sure is annoying.

**Comment by mslipper** [comment]
This happens because OSX requires all binaries to be notarized by the Apple mothership before they'll run on Mac. The workaround here is to right click on the binary, then select "Open." I'll update the docs to highlight this - there may also be a way to notarize/avoid this prompt if we use Homebrew, which I'll look into.

**Comment by tessr** [comment]
Yeah, I was able to get around it, but it would be nice to get this signed properly to avoid this prompt. Again, very low priority but nice to have.
